model_bakery/generators.py

```diff
@@ -60,17 +61,26 @@
     CITextField = None
 
 
+def _make_integer_gen_by_range(field_type):
+    min_int, max_int = BaseDatabaseOperations.integer_field_ranges[field_type.__name__]
+
+    def gen_integer():
+        return random_gen.gen_integer(min_int=min_int, max_int=max_int)
+
+    return gen_integer
```

Comment:
Loved this solution! I didn't know this `BaseDatabaseOperations` class and it makes total sense to replace the range to use it.